android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/WebBrowserActivity.java

```diff
@@ -323,7 +323,10 @@ public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
           try {
             new URL(urlStr);
           } catch (MalformedURLException e) {
-            KMLog.LogException(TAG, "", e);
+            // Intentionally not logging "No protocol" exception because it may be a query
+            if (e != null && !e.getMessage().startsWith("no protocol:")) {
+              KMLog.LogException(TAG, "", e);
```

Comment:
This should never be logged as an exception (`KMLog.d()` would be okay) -- no matter what the format. We don't want exceptions logged for user input, only for errors in code.